<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>i10gui</class>
 <widget class="QMainWindow" name="i10gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>i10chic_gui</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="kplusButton">
      <property name="text">
       <string>K3 +</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>H BPM1 +</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>H BPM2 +</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="bumpleftplusButton">
      <property name="text">
       <string>BUMP LEFT +</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>BUMP RIGHT +</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="kminusButton">
      <property name="text">
       <string>K3 -</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>H BPM1 -</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>H BPM2 -</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="bumpleftminusButton">
      <property name="text">
       <string>BUMP LEFT -</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>BUMP RIGHT -</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <layout class="QVBoxLayout" name="matplotlib_layout"/>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="quitButton">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStuff">
   <property name="text">
    <string>Stuff</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
